pytorch_pfn_extras/training/extensions/__init__.py

```diff
@@ -7,6 +7,7 @@
 from pytorch_pfn_extras.training.extensions.micro_average import MicroAverage  # NOQA
 from pytorch_pfn_extras.training.extensions.print_report import PrintReport  # NOQA
 from pytorch_pfn_extras.training.extensions.progress_bar import ProgressBar  # NOQA
+from pytorch_pfn_extras.training.extensions.progress_bar_notebook import ProgressBarNotebook  # NOQA
```

Comment:
This might add additional software dependency to IPython or ipywidgets.
Is it better to avoid that and we should import it as
`from pytorch_pfn_extras.training.extensions.progress_bar_notebook import ProgressBarNotebook as ProgressBar` ??

Reply:
Lets wrap it in a try/except

Reply:
Wrapped by try except